src/app/app.module.ts

```diff
@@ -1,18 +1,17 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
+import { FormsModule } from '@angular/forms'; // <-- import this module
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { CreateButtonComponent } from './components/create-button/create-button.component';
+import { AddCardComponent } from './components/add-card/add-card.component';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
-  ],
+  declarations: [AppComponent, CreateButtonComponent, AddCardComponent],
+  imports: [BrowserModule, AppRoutingModule, ReactiveFormsModule, FormsModule],
   providers: [],
-  bootstrap: [AppComponent]
+  bootstrap: [AppComponent],
```

Comment:
Please config the `eslint` if you want or not to keep a comma on the last element in the object for consistency. I see in some places the comma is available and in some place, it is not.

Reply:
done!

Reply:
done!